traffic_ops/app/lib/API/User.pm

```diff
@@ -277,6 +285,7 @@ sub current {
 			@data, {
 				"id"              => "0",
 				"username"        => $current_username,
+				"tenantId"	  => $self->current_user_tenant(),
```

Comment:
I assume the tenancy of an LDAP user should be set by the operator. He might want it to be the single tenant in the system (if he has only one tenant). He might want it to be the "root" tenant. And he might want it to be "undef" which is a powerless tenant (such like a read-only user).

If you agree with the above I think we need to provide a way to specify the "LDAP" users' tenant when the tenancy feature is first activated. Until then (and if not set) an "undef" is assumed. 

What would you say about adding a property in the tenants table, allowing to mark the "LDAP" tenant?